protocol/avdl/keybase1/wot.avdl

```diff
@@ -36,8 +45,8 @@ protocol wot {
   void wotVouchCLI(int sessionID, string assertion, string vouchText, Confidence confidence);
 
   enum WotReactionType {
-    ACCEPT_0,
-    REJECT_1
```

Comment:
I had written a bug that didn't pass this value. Typescript didn't catch it. It's too easy for the golang default value to conjure itself. This way, rejection is the default which seems safer.

Reply:
interesting. so, if you reject, then it's gone. if you accept, and that was a fat finger or bug somehow, then you can still reject it. from a user perspective, if we do screw this up somehow, it might be easier for them if accept is the default.

Reply:
Let's have no more bugs. I realize I'm arguing both sides here. If we have a bug then don't sign anything 'positive' on behalf of the user.

Reply:
Could add a third NONE_0, ACCEPT_1, REJECT_2 where NONE errors if you'd like.

Reply:
i'm not going to the mat on this. but yes, i think if safety is the goal, then a NONE_0 would be the safest.